apps/app/pages/index.tsx

```diff
@@ -93,8 +176,71 @@ function Dashboard() {
                 title="Tens a certeza que queres notificar?"
                 cancelText="Não"
                 okText="Sim"
-                onConfirm={(_) => notify_selected_ninjas()}
+                onConfirm={(_) => {
+                  notify_selected_ninjas();
+                  setConfirmed_selected(true);
+                }}
+                disabled={confirmed_selected}
               >
+                <Modal
+                  visible={visible_selected}
+                  onCancel={handleCloseModal_selected}
+                  centered={true}
+                  closable={false}
+                  title="Lista de Emails"
+                  footer={[
+                    <Button
+                      key="close"
+                      type="primary"
+                      onClick={handleCloseModal_selected}
+                    >
+                      Close
+                    </Button>,
+                  ]}
+                >
```

Comment:
This whole `Modal` could be wrapped into a component (with some props) and then be rendered after an organizer clicks any of the buttons. This way, the code will be cleaner and more reusable 💪🏼